patches/api/0373-Makes-it-possible-to-change-the-xPush-and-zPush-of-p.patch

```diff
@@ -0,0 +1,38 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: EpicKnarvik97 <kristian.knarvik@knett.no>
+Date: Sat, 5 Mar 2022 13:57:06 +0100
+Subject: [PATCH] Makes it possible to change the xPush and zPush of powered
+ minecarts
+
+Adds a method for getting the push of a powered minecart as a two-dimensional vector where the first coordinate is xPush and the second is zPush.
+Adds a method for setting the push of a powered minecart that expects a two-dimensional vector where the first coordinate is xPush and the second is zPush.
+
+diff --git a/src/main/java/org/bukkit/entity/minecart/PoweredMinecart.java b/src/main/java/org/bukkit/entity/minecart/PoweredMinecart.java
+index 2076af6c029027d3243ebfa518f4d69b716de7fe..4c49fb577714e1654752e3bed61579c16f55cef9 100644
+--- a/src/main/java/org/bukkit/entity/minecart/PoweredMinecart.java
++++ b/src/main/java/org/bukkit/entity/minecart/PoweredMinecart.java
+@@ -21,4 +21,22 @@ public interface PoweredMinecart extends Minecart {
+      * @param fuel Number of ticks until the minecart runs out of fuel
+      */
+     public void setFuel(int fuel);
++    // Begin paper - Adds some extra methods to interact with PoweredMinecart axis thrust ("Push").
++
++    /**
++     * Get the push (xPush, zPush) of the minecart.
++     *
++     * <p>Return a 2-dimensional vector x, z</p>
++     *
++     * @return The push (xPush, zPush) of the minecart.
++     */
++    public double[] getPush();
++
++    /**
++     * Set the push (xPush, zPush) of the minecart.
++     *
++     * <p>Push should be a 2-dimensional vector x, z</p>
++     *
++     * @param push The push (xPush, zPush) of the minecart.
++     */
++    public void setPush(double[] push);
++    // End paper
```

Comment:
Please read the contribution guide on how to write proper paper comments.
https://github.com/PaperMC/Paper/blob/master/CONTRIBUTING.md